app/page.tsx

```
@@ -200,338 +200,338 @@ TrustLogoStrip.displayName = 'TrustLogoStrip';
-                    <div className="flex items-center gap-1 rounded-full border border-accent/30 bg-accent/20 px-3 py-1 text-sm backdrop-blur-glass">
-                      <TrendingUp className="h-4 w-4 text-accent" />
-                      <span>10X ROI Guarantee</span>
+              <ProgressiveBackgroundLoader
+                variant="hero"
+                className="absolute inset-0 z-0 h-full w-full bg-background bg-[radial-gradient(ellipse_60%_80%_at_20%_40%,rgba(139,92,246,0.1),transparent)] bg-[radial-gradient(ellipse_80%_80%_at_50%_-20%,rgba(29,155,240,0.1),rgba(255,255,255,0))]"
+                fallbackStyle={{
+                  background:
+                    'linear-gradient(to bottom, hsl(var(--background)), hsl(var(--card))',
+                }}
+              >
+                <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)